.github/workflows/test.yml

```diff
@@ -50,6 +50,7 @@ jobs:
       - name: Test
         run: |
           yarn install --immutable
+          yarn sync
```

Comment:
Tests and lint were failing without it. I think the build step creates the needed types for the .env, and upload artifact doesn't transfer that over?

Reply:
@jgerigmeyer  How did you fix this?

Reply:
Oh, right. I think the test failure was unrelated, and the lint command is supposed to run `sync` internally anyway, but there was a typo and it was running in parallel instead of synchronously. Fixed here: https://github.com/oddbird/oddcontrast/pull/206/commits/a70f6b42766e6920a2c76c2b19929971e5d7bf66#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R26